packages/angular_devkit/schematics/tools/file-system-engine-host-base.ts

```diff
@@ -108,6 +125,14 @@ export abstract class FileSystemEngineHostBase implements
       return null;
     }
 
+    if (partialDesc.extends) {
+      return {
```

Comment:
We probably want to fallback to the rest of the function, so people can extend and change some argument like `factory`.